verticapy/learn/metrics.py

```diff
@@ -757,6 +757,7 @@ def auc(
     input_relation: Union[str, vDataFrame],
     cursor=None,
     pos_label: (int, float, str) = 1,
+    nbins: int = 10000,
```

Comment:
Why does the default value of nbins need to be so big? Usually, a default value should be optimal for the most common cases not the extreme cases.

Reply:
I got too many feedback that the AUC is not precise. Many use-cases are using unbalanced datasets and the AUC result for that case is not precise enough.